src/main/java/Tommi.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class Tommi {
+    private static final int MAX_TASKS = 100;
+    private static final String[] tasks = new String[MAX_TASKS];  // Array to store tasks
+    private static final boolean[] isCompleted = new boolean[MAX_TASKS];  // Array to store task completion status
+    private static int taskCount = 0;  // Counter to keep track of the number of tasks
+
+    public static void main(String[] args) {
+        printIntroMessage();
+
+        Scanner scanner = new Scanner(System.in);
+        String input;
+
+        while (true) {
+            input = scanner.nextLine(); // Read user input
+
+            if (input.equals("bye")) {
+                printExitMessage();
+                break; // Exit the loop
+            } else if (input.equals("list")) {
+                listTasks();
+            } else if (input.startsWith("mark ")) {
+                int taskIndex = Integer.parseInt(input.substring(5)) - 1;
+                markTask(taskIndex);
+            } else if (input.startsWith("unmark ")) {
+                int taskIndex = Integer.parseInt(input.substring(7)) - 1;
+                unmarkTask(taskIndex);
+            } else {
+                addTask(input);
+            }
+        }
+    }
+
+    private static void printIntroMessage() {
+        String intro =
+                          " ______                   \n"
+                        + "/_  __/__  __ _  __ _  (_)\n"
+                        + " / / / _ \\/  ' \\/  ' \\/ / \n"
+                        + "/_/  \\___/_/_/_/_/_/_/_/  \n"
+                        + "____________________________________________________________\n"
+                        + "Hello! I'm Tommi!\n"
+                        + "What can I do for you?\n"
+                        + "____________________________________________________________\n";
```

Comment:
You can consider using 8 spaces to split up your line as stated in the coding standard